src/components/pages/episodes-page/episodes-toggle/EpisodesToggle.constants.tsx

```diff
@@ -0,0 +1,4 @@
+export enum EpisodesToggleOption {
+  ASC = 'asc',
+  DESC = 'desc',
+}
```

Comment:
You said you don't like. How would you like to see it?

```ts
export enum HttpVerbs {
  Delete = 'DELETE',
  Get = 'GET',
  Patch = 'PATCH',
  Post = 'POST',
  Put = 'PUT',
}
```

Reply:
@scottbromander sometimes you just have to follow what the standard is already `PascalCase`:

https://www.typescriptlang.org/docs/handbook/enums.html

Sure I could see ALL_CAPS for single constant variables but not for enums.